server/test/auth/register.spec.js

```diff
@@ -0,0 +1,59 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const app = require("../../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+const expect = chai.expect;
+
+describe("POST /register", () => {
+  describe("When parameters are invalid", () => {
+    let user = {
+      name: "John Doe",
+      email: "fakeEmail@gamil.com",
+      password: "123"
+    }
+
+    it("it returns 400 status code", (done) => {
+      chai
+        .request(app)
+        .post("/register")
+        .send(user)
+        .end((err, res) => {
+          res.should.have.status(400);
+          done();
+        });
+    });
+  });
+
+  describe("When parameters are valid", () => {
+    let user = {
+      name: "John Doe",
+      email: "fakeEmail@gamil.com",
+      password: "123",
+    };
+
+    it("it returns 200 status code", (done) => {
+      chai
+        .request(app)
+        .post("/register")
+        .send(user)
+        .end((err, res) => {
+          res.should.have.status(200);
```

Comment:
We should actually be checking if the information was added to the database. 

We also shouldn't be adding directly to our database in tests - take a look at:
- https://github.com/nodkz/mongodb-memory-server
- https://medium.com/javascript-in-plain-english/how-i-setup-unit-test-for-mongodb-using-jest-mongoose-103b772ee164 (Don't need to use jest, but use it to get the idea)